ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.ts

```diff
@@ -45,6 +51,26 @@ const resetTrio = (data: Data, trio: string) => {
   data.megaStack = [];
 };
 
+const dirs = ['dirN', 'dirNE', 'dirE', 'dirSE', 'dirS', 'dirSW', 'dirW', 'dirNW'];
```

Comment:
I think `dirs` is a little too ambiguous of a name for a top level variable.  This is really a map of dir numbers to dir names, so `dirNames` is fine-ish, but `dirNumToName` maybe even more clear elsewhere.